compiler/include/byteir/Dialect/mhlo/Util/CustomCallUtil.h

```diff
@@ -69,6 +69,10 @@ constexpr llvm::StringRef getOneHotName() {
   return CUSTOM_CALL_NAME_PREFIX "one_hot";
 }
 
+constexpr llvm::StringRef getReshapeName() {
+  return CUSTOM_CALL_NAME_PREFIX "reshape";
```

Comment:
Is it necessary to add `byteir.reshape`?

Reply:
There are four reasons for switching to byteir.reshape instead of retaining the original tf.Reshape form:
1. If possible, convert regular op (such as reshape op) to custom_call@byteir.***, rather than custom_call@tf.***
2. custom_call@byteir.*** indicates that it cannot be converted to regular mhlo op, but we have identified this case. But the form custom_call@tf.*** does not convey this meaning
3. when other front-end reshape op appears and cannot be converted into regular mhlo.reshape op and mhlo.dynamic_reshape op, it can also be converted into byteir.reshape, a unified representation
4. getReshapeName function will be used in shape/bounded shape inference